prov/efa/src/efa_av.h

```diff
@@ -30,36 +30,6 @@
  * SOFTWARE.
  */
 
-#include "config.h"
-
-#include <asm/types.h>
-#include <errno.h>
-#include <fcntl.h>
-#include <arpa/inet.h>
-#include <netinet/in.h>
-#include <poll.h>
-#include <stdio.h>
```

Comment:
There changes probably should be with the previous comment when `efa_av.h` was introduced.

Reply:
done